pay-api/src/pay_api/services/fee_schedule.py

```diff
@@ -284,6 +284,7 @@ def find_by_corp_type_and_filing_type(  # pylint: disable=too-many-arguments
             **kwargs
     ):
         """Calculate fees for the filing by using the arguments."""
+        from pay_api.services import flags  # pylint: disable=import-outside-toplevel
```

Comment:
If I didn't put the import here, ran into circular dependency troubles or flags not having `is_on`